va/validation-authority.go

```diff
@@ -333,6 +333,30 @@ func (va *ValidationAuthorityImpl) validateSimpleHTTP(identifier core.AcmeIdenti
 		return challenge, err
 	}
 
+	contentTypes, ok := httpResponse.Header[http.CanonicalHeaderKey("content-type")]
```

Comment:
Just use "Content-Type" (it's extremely unlikely to change). Or make a constant that is the return value of CanonicalHeaderKey.